tests/e2e/lifecycle/test_no_liveness_probe.py

```diff
@@ -0,0 +1,71 @@
+# BZ2142901 Automated Test
```

Comment:
Lifecycle may not be a relevant place for this test, should it be placed under z_cluster since it is a cluster-level pod check?
@prsurve wdyt?

Reply:
Yes test can be moved to brown squad